actix-web/README.md

```diff
@@ -1,5 +1,7 @@
-# Actix Web with shuttle
```

Comment:
This readme should stay like it was, since it explains some oddities of using actix with shuttle.